sdk/identity/Azure.Identity/src/ImdsManagedIdentityProbeSource.cs

```diff
@@ -0,0 +1,196 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Microsoft.Identity.Client.AppConfig;
```

Comment:
Is this namespace needed for anything?
It contains `ManagedIdentityId` which doesn't appear to be used in this file.

Reply:
nope